src/components/views/messages/MStickerBody.js

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
This file is a verbatim copy of MImageBody with the names changed and the copyright updated. There **must** be a better way to do it than this.

Reply:
Agreed. This should extend the MImageBody and otherwise just be empty for future stuff.

Reply:
Good point. I've implemented it as suggested.